SPARCore/GADPublication.m

```diff
@@ -0,0 +1,53 @@
+#import "GADPublication.h"
+
+@implementation GADPublication
+
+static NSString *const API_PUBLICATION_PATH = @"publications";
+static NSString *const API_ARTICLE_PATH = @"articles";
```

Comment:
`API_ARTICLE_SUFFIX` or something? I'm trying to differentiate these from the API keys